crates/cargo-test-support/src/lib.rs

```diff
@@ -737,12 +864,18 @@ impl Execs {
     ///
     /// See [`compare`] for supported patterns.
     ///
-    /// Care should be taken when using this method because there is a
-    /// limitless number of possible things that *won't* appear. A typo means
-    /// your test will pass without verifying the correct behavior. If
-    /// possible, write the test first so that it fails, and then implement
-    /// your fix/feature to make it pass.
-    #[deprecated]
```

Comment:
Not going to block this, but still prefer them being deprecated, as people have less change to misuse.

Reply:
If you have other ideas, I'm open